### PR TITLE
Added entry 2023:orlando.benacchio:imex

### PR DESCRIPTION
**Pull request by GiuseppeOrlando878776**
New entry of a published article. The preprint version is present in the 'publications-2022.bib' file, so I do not know if this is the correct way to update it

### CODE REVIEW

File: publications-2023.bib [hunk 1]
```diff
@@ -155,6 +155,19 @@ @Article{2023:orlando:filtering
...
+@Article{2023:orlando.benacchio:imex,
+  author    = {Orlando, Giuseppe and Benacchio, Tommaso and Bonaventura, Luca},
+  title     = {An {IMEX-DG} solver for atmospheric dynamics simulations with adaptive mesh refinement},
+  journal   = {Journal of Computational and Applied Mathematics},
+  pages     = {115124},
+  year      = {2023},
+  month     = {aug},
+  volume    = {427},
+  doi       = {10.1016/j.cam.2023.115124},
+  url       = {https://www.sciencedirect.com/science/article/pii/S0377042723000687}
```
Comment (marcfehling) [inline publications-2023.bib:167]:
```suggestion
  url       = {https://www.sciencedirect.com/science/article/pii/S0377042723000687},
```

**Review by marcfehling — APPROVED** [review]
Thank you for submitting your publication to the list! :)

We already have the preprint in our database. Could you remove the preprint in your pull request, and add a line `preprint = {https://arxiv.org/abs/2210.07898}` to the 2023 entry? Thank you!
https://github.com/dealii/publication-list/blob/793e8d8664c1370d8702dd5f6196d276f0b78687/publications-2022.bib#L932-L940

File: publications-2023.bib (hunk 1 above)
Comment (GiuseppeOrlando878776) [reply publications-2023.bib]: I should have not properly modified the files. Best

**Review by marcfehling — APPROVED** [review]

**Pull request merged**